components/VirtualScrollView.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {FlatList} from 'react-native';
+
+export const VirtualScrollView = props => {
+  return (
+    <FlatList
+      style={props.style}
+      contentContainerStyle={props.contentContainerStyle}
+      data={[]}
+      ListEmptyComponent={null}
+      keyExtractor={() => 'dummy'}
```

Comment:
@khamui what does the dummy thing do?

Reply:
oh just saw this now @phillimorland , because we don't have any data this keyExtractor does nothing here.

Reply:
danke :)